compiler/quilt/tools/command.py

```diff
@@ -68,7 +69,18 @@
 
 LOG_TIMEOUT = 3  # 3 seconds
 
-VERSION = pkg_resources.require('quilt')[0].version
+try:
+    VERSION = pkg_resources.require('quilt')[0].version
+except pkg_resources.DistributionNotFound:
+    # Not installed, so we'll get the version manually.
```

Comment:
So I still don't understand: why is this needed? When would quilt not be installed?

Reply:
@eode wrote:
> It is super-hacky, but at the time it was written, the goal was to enable invoking main.py from the tests directory so that pytest runs main.py, without an install, and without creating any new files -- although, some of those restrictions may have been made in part simply as questions on the job description.

To be more clear:  I was instructed to do so.  It was one of the requirements I was initially given on this specific project.  

However, it may be a requirement that has has since changed, like the requirement on not creating any new files has (a `VERSION` file was permitted).

..in either case, it was what I was instructed to do, and I found a way to do it with the specified restrictions.  If you want that capacity removed, or if you'd like for me to use a different implementation, I'll follow the new requirements you set forth.

As to when quilt wouldn't be installed:  I think it was requested of me in order to fulfil a specific (or potential) testing or CI setup need, but I don't know the reason definitively.

Reply:
This was me -- at the time, I was cautious about requiring pip install to be run before unittests can pass... but more & more, I'm seeing that pip install is kinda required for even basic development.

So if you want to require this, I'm ok with it.  @dimaryaz gets the final say, of course.

Reply:
..sounds good.

Reply:
Yeah, let's require quilt to be installed. We already need all of the requirements to be installed - so we're not saving anything by not requiring quilt itself.

Reply:
Sounds good, I'll excise that code.